react/features/invite/components/DialInNumbersForm.js

```diff
@@ -1,7 +1,7 @@
 import Button from '@atlaskit/button';
 import { StatelessDropdownMenu } from '@atlaskit/dropdown-menu';
 import { FieldText } from '@atlaskit/field-text';
-import ExpandIcon from '@atlaskit/icon/glyph/expand';
+import ExpandIcon from '@atlaskit/icon/glyph/editor/expand';
```

Comment:
This icon hasn't been introduced in this PR and it's my fault that I have missed it when reviewing, but we need to stick to the font icons for now for the reason mentioned above.